### ISSUE TITLE
Legacy A4Tech Mice

### ISSUE THREAD

**Issue by moriel5** [maxmati/a4tech-bloody-linux-driver#33]
Since it appears that A4Tech may be reusing code for different mice, would it theoretically be possible to add support for, say, the BW-35 or the RBW-5?

**Comment by C0rn3j** [comment]
See if https://gitlab.com/C0rn3j/a4tech_bloody_p85_driver/ doesn't already support it

**Comment by moriel5** [comment]
Thanks, though it seems like it does not.
I'll send the PID in the evening, after I return home, and check how to obtain the firmware when I have the time (probably next week).

**Comment by C0rn3j** [comment]
Oh so this is legacy-legacy, not bloody? Yeah that won't work unfortunately

**Comment by moriel5** [comment]
Correct, however while I forget the details, since several years have elapsed since I checked things, I do believe that from the pre-Bloody legacy A4Tech mice, these may just be recent enough (or at least, has recent enough official driver support, that things may be similar enough to add limited support (remapping buttons, changing DPI and viewing battery level).

**Comment by moriel5** [comment]
Looks like the PID of the RBW-5 is 001a (wirelessly connected via a USB dongle)
When I have the time, I'll look for my BW-35 (even more basic, and my unit is PS/2).

**Comment by C0rn3j** [comment]
PID is not much useful - there does not seem to be any mention of those across the bloody software either - you'd have to write your own driver - i.e. bust out Wireshark to get the communication protocol and go from there.

Reversing the original (Oscar?) software would probably also help.

LLMs could possibly help there.
